jcommune-model/src/main/java/org/jtalks/jcommune/model/entity/PostComment.java

```diff
@@ -43,6 +43,8 @@
     private Post post;
     private Map<String, String> attributes = new HashMap<>();
     private DateTime deletionDate;
+    private DateTime modificationDate;
```

Comment:
Please add (or modify existing) tests for DAO (that store the data and check if everything is saved) to accommodate these new fields.